upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -127,6 +127,10 @@ func buildNodeupTags(role api.InstanceGroupRole, cluster *api.Cluster, clusterTa
 		tags.Insert("_networking_cni")
 	}
 
+	if networking.AmazonVPCIPVlan != nil {
```

Comment:
I hooked the cni conflist template into the Loader with this tag. I'm not sure if that is the right place for it but it seemed to follow established patterns in the code.

Reply:
FYI: We're trying to get rid of tags in general, because a function tends to be clearer.  I don't think this actually does anything (it looks like you're just using it on a resource, where I don't _think_ tags act as a filter, but I could very well be wrong)

Reply:
The treewalker ignores tag directories that are [unknown](https://github.com/kubernetes/kops/blob/88984d4b477dbb59e124f0626e1cc23d9b5b0aef/upup/pkg/fi/loader/tree_walker.go#L119-L122) 

I spent a fair bit of time looking at how templates get handled in the code and this seemed to be the way it's done in nodeup, though there do appear to be other ways in other places. Is there another example of doing templates in nodeup without the tree walker / tags that I've overlooked and could emulate?